.travis.yml

```
@@ -1,27 +1,41 @@
+## Documentation: http://docs.travis-ci.com/user/languages/julia/
+
+language: julia
+
 branches:
   only:
   - master
-  - staging
-  - trying
   - /^release\/.*$/ # release branches
   - /^v[0-9]+\.[0-9]+\.[0-9]+$/ # version tags
-language: julia
+
 os:
   - linux
   - osx
   - windows
+
+arch:
+  - amd64
+  - i686
+
 julia:
   - "1.0"
-  - "1.2"
-  - "1.3"
+  - "1" # currently 1.5
```

Won't that just get outdated? Perhaps like this:
```suggestion
  - "1" # refers to the latest 1.x release
```

---

That's precisely the intent. Comments shouldn't just restate the code, but should add value. In this case, it gives knowledge of what it was testing with at the time when it was merged.